frontend/src/routes/Search/SearchResults/utils.tsx

```diff
@@ -267,9 +267,12 @@ export function getRowActions(
     id: 'startVM',
     title: t('Start {{resourceKind}}', { resourceKind }),
     click: (item: any) => {
+      const path = item?._hubClusterResource
+        ? `/apis/subresources.kubevirt.io/v1/namespaces/${item.namespace}/virtualmachines/${item.name}/start`
+        : `/virtualmachines/start`
```

Comment:
I'm thinking this could be simplified and also made more generic if we did something like this.

```
const path = item?._hubClusterResource
        ? `/apis/subresources.kubevirt.io/v1/namespaces/${item.namespace}/virtualmachines/${item.name}/start`
        : `/proxytocluster?path=/apis/subresources.kubevirt.io/v1/namespaces/${item.namespace}/virtualmachines/${item.name}/start`
```

Then on the backend you don't need to rebuild the path.
It also makes the backend closer to being generic.

**Maybe this change should be separate from this PR.**

Reply:
Yes, this would be nice, would require some updates to how we get the cluster proxy route and token as well. Certainly work for another PR in my opinion.

Reply:
Thanks Zack! We are at the finish line, so I'm good if we push this refactor to 2.13. I just want to make sure we establish the correct pattern.